documentation/userguide.md

```diff
@@ -460,6 +470,34 @@ match: {
 }
 ```
 
+
+## Example: Profile and upload multiple applications that are started from the same process
+
+Some applications have a plugin-like architecture, this means that you have one host process that dynamically starts an application.
+Examples are IIS, Windows or COM+ services.
+Distinguishing traces solely on the host process is not possible as it is the same.
+To mitigate this uploading may be configured using `loadedAssemblyPathRegex`, e.g. by uploading all coverage that stems from traces that loaded assemblies matching `C:\webapp-1\.*` to one project and loaded assemblies matching `C:\webapp-2\.*` to another project.
+
+Please note that the profiler process may _not_ (yet) be turned on/off using `loadedAssemblyPathRegex`.
+
+```yaml
+match:
+  - profiler:
+      targetdir: C:\profiler\traces
+      enabled: true
+  - loadedAssemblyPathRegex: 'C:\webapp-1\.*'
```

Comment:
I think these regexes should be `'C:\\webapp-1\\.*'` and `'C:\\webapp-2\\.*'` otherwise the backslash accidentally escapes the dot.